code-challenges.test.js

```diff
@@ -56,4 +143,48 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("firstVowels", () => {
+    it("returns log of the index of the first vowel", () => {
+      expect(firstVowels(vowelTester1)).toEqual(1)
+      expect(firstVowels(vowelTester2)).toEqual(0)
+      expect(firstVowels(vowelTester3)).toEqual(2)
+    })
+  })
+
+//   ReferenceError: firstVowels is not defined
+
+//   149 | describe("firstVowels", () => {
+//   150 |     it("returns log of the index of the first vowel", () => {
+// > 151 |       expect(firstVowels(vowelTester1)).toEqual("1")
+//       |       ^
+//   152 |       expect(firstVowels(vowelTester2)).toEqual("0")
+//   153 |       expect(firstVowels(vowelTester3)).toEqual("2")
+//   154 |     })
+
+//   at Object.expect (code-challenges.test.js:151:7)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
 // b) Create the function that makes the test pass.
+
+const firstVowels = (string) => {
+    let vowels = ["a", "e", "i", "o", "u"]
+    for (let i=0; i < string.length; i++) {
+        if (vowels.includes(string[i])) {
+            return i
+        }
+    }
+}
+
+//*Feedback Request* I had an especially difficult time with this one.  It was explained to me, but I'm still not quite grasping it.  I understand that in order to find the index, I'll need to first change the string into an array.  I set a variable in order to isolate the vowels.  Using For Loops, I search the length of the string.  Setting the if statement, I use the vowels.includes method to look in the string placeholder and its index.  I want just the index returned.  I feel like I'm missing something because it just isn't clicking. Shouldn't it spit out all of the vowels/indexes? Where did I single out only the first vowel of the string?
+
```

Comment:
Strings are zero indexed as well and for this situation, you don't need to change it to an array. Your conditional statement is the reason you are only getting the first index of a vowel. As soon, as that condition is true. The for loop will stop. One of the way you can see this is by adding a console.log(i) in between the for loop and conditional and testing out a string with a vowel in the middle and a string with a vowel at the end. Pay attention to the difference in the console.